docs/osf-trivia-list/dasharo.md

```diff
@@ -0,0 +1,58 @@
+# What is Dasharo?
+
+Dasharo is registered trademark and product developed by [3mdeb](https://3mdeb.com).
+
+Dasharo is an open-source firmware distribution focusing on
+[seamless deployment](#dasharo-seamless-deployment),
+[clean and simple code](#dasharo-clean-and-simple-code),
+[long-term maintenance](#dasharo-long-term-maintenance),
+[professional support](#dasharo-support-package),
+[transparent validation](#dasharo-transparent-validation),
+[superior documentation](../index.md)
```

Comment:
We need section that proves that fact. I saw some community members claiming that our documentation is very good, but I would go to fact that we are strict about providing extending explanation using very known pattern described [here](https://github.com/Dasharo/docs#navigation-menu)

Reply:
> We need section that proves that fact. I saw some community members claiming that our documentation is very good

Ooof, now I wish I had those messages saved somewhere.

Reply:
Wasn't sure which adjectives to use so went with `extensive and structured documentation` and the link you provided